hack/release/Makefile

```diff
@@ -10,6 +10,21 @@ release: .vm.created
 	./remote-execute.sh "cd calico && git fetch --tags && git checkout $(GIT_BRANCH)"
 	./remote-execute.sh "cd calico && make release"
 
+git-fetch-remote: .vm.created
+	./remote-execute.sh "cd calico && git fetch --tags && git checkout $(GIT_BRANCH)"
+
+update-versions-remote: .vm.created
+	./remote-execute.sh "cd calico && make update-versions"
+
+generate-manifests-remote:
```

Comment:
Also want .vm.created here

Reply:
No longer needed as we aren't using GCP VMs anymore.